DESCRIPTION

```diff
@@ -91,5 +92,8 @@ Config/testthat/edition: 3
 Suggests: 
     testthat(>= 3.0.0), 
     scRNAseq,
-    irlba
+    irlba,
+    cellxgene.census,
+    zellkonverter
```

Comment:
if HPCell uses zellkonverter to read it should be in the import. Unless I am missing something.